src/translations/en.json

```diff
@@ -176,5 +176,7 @@
     "moveElementNotAllowedError": "You cannot move this element to the targeted folder. Unauthorized action",
     "notElementCreator": "You are not the element's creator",
     "uploadingElement": "Upload in progress: no operation permitted",
+    "invalidFormatOrName" : "Invalid format or name",
```

Comment:
Why 2 different messages `invalidFormatOrName` and `incorrectFileError` ?

Reply:
in fact when i merge main into my branch there was another message like this
i will use only one